### DISCUSSION TITLE
How to quey or view data in the containers?

### DISCUSSION THREAD

**Discussion by danielniccoli** [Azure/Azurite#2363] [Q&A]
I'm using Azurite (VS Code extension) to develop an Azure Durable Functions application locally.

I want to understand how Durable Functions persisted event data to the containers in certain scenarios. If I would develop with Azure Cosmos DB, I would have a nice UI where I could browse the data or send requests to. With the old Cosmos DB emulator I would have the same thing. With Azurite, I have not found a similar thing.

Is there a UI or a client that I could use to send requests to Azurite to view the data?

**Comment by blueww** [comment]
@danielniccoli 

You can use storage explorer to view the stored data in Azurite or send request to Azurite, very similar from connect storage explorer to real public Azure Storage server. 
See more details in https://learn.microsoft.com/en-us/azure/storage/common/storage-explorer-emulators
